src/main/java/frc/trigon/robot/constants/CameraConstants.java

```diff
@@ -25,6 +25,22 @@ public class CameraConstants {
                     new Rotation3d(Math.PI - Units.degreesToRadians(0), Units.degreesToRadians(-33), Math.PI + Units.degreesToRadians(0))
             );
 
+    private static final int
+            CAMERA_WIDTH = 1600,
+            CAMERA_HEIGHT = 1200,
+            CAMERA_FPS = 60,
+            AVERAGE_CAMERA_LATENCY_MILLISECONDS = 35,
+            CAMERA_LATENCY_STANDARD_DEVIATIONS_MILLISECONDS = 5;
+    private static final Rotation2d CAMERA_FOV = Rotation2d.fromDegrees(90);
+    private static final double
+            AVERAGE_PIXEL_ERROR = 0.25,
+            PIXEL_STANDARD_DEVIATIONS = 0.08;
+    public static final SimCameraProperties SIM_CAMERA_PROPERTIES = new SimCameraProperties();
+
+    static {
+        configureSimCameraProperties();
+    }
```

Comment:
This should be after var declaration

Reply:
It needs to be done before because it is used for the following variables

Reply:
It's not used for the Object detection camera

I meant to write this here